fe/fe-core/src/main/java/com/starrocks/sql/optimizer/rewrite/ScalarOperatorEvaluator.java

```diff
@@ -169,18 +183,25 @@ public ScalarOperator evaluation(CallOperator root) {
             return operator;
         } catch (AnalysisException e) {
             LOG.debug("failed to invoke", e);
+            throw new StarRocksPlannerException(ErrorType.USER_ERROR, e.getMessage());
```

Comment:
don't throw exception, some FE function can't resolve but BE can do it

Reply:
fixed